array-manipulation-2/src/take.js

```diff
@@ -0,0 +1,18 @@
+/* exported take */
+function take(array, count) {
+  var output = [];
+  for (var i = 0; i < array.length; i++) {
+    if (i <= (count - 1)) {
+      output.push(array[i]);
+    }
```

Comment:
Instead of looping from start to finish of the array, think about where you actually want to loop from/to.